models/forms/LoginForm.php

```diff
@@ -46,7 +46,7 @@ public function validatePassword($attribute, $params)
         if (!$this->hasErrors()) {
             $user = $this->getUser();
 
-            if (!$user || !$user->validatePassword($this->password)) {
+            if (!$user || !$user->validatePassword($this->password) || $user->password_hash === null) {
```

Comment:
You must check if the password is null before validating it, otherwise it's useless...
`if (!$user || $user->password_hash === null || !$user->validatePassword($this->password))`